packages/storybook8/stories/INTERNAL/CaptionsSettingsModal/Docs.mdx

```diff
@@ -0,0 +1,16 @@
+import { Canvas, Meta, ArgTypes } from '@storybook/blocks';
+import { _CaptionsSettingsModal } from '@internal/react-components';
+import * as CaptionsSettingsModalStories from './index.stories';
+import CaptionsSettingsModalExampleText from '!!raw-loader!./snippets/CaptionsSettingsModal.snippet.tsx';
+
+<Meta of={CaptionsSettingsModalStories} />
+
+# CaptionsBanner
+
+A component for displaying a CaptionsBanner with user icon, displayName and captions text.
+
+<Canvas of={CaptionsSettingsModalStories.CaptionsSettingsModal} source={{ code: CaptionsSettingsModalExampleText }} />
```

Comment:
This canvas causes the modal to pop up and take up the whole screen...
![image](https://github.com/user-attachments/assets/f2a551a9-3c22-4494-8494-735e52d8f7d3)

Reply:
Note that the modal opens up as soon as you click on the doc and covers the whole screen.

Reply:
Yeah, it is the same in storybook6 https://60c7ae6891f0e90039d7cd54-mlgfwylnxa.chromatic.com/, I think we need to add a button to open modal on click. @carocao-msft was it intended behaviour or should we modify this?

Reply:
We should modify this

Reply:
Done, added the button